gui/slick/js/rootDirs.js

```diff
@@ -181,8 +181,8 @@ $(document).ready(function() {
     });
 
     $('#defaultRootDir').on('click', function(){
-        if ($("#rootDirs option:selected").length) {
-            setDefault($("#rootDirs option:selected").attr('id'));
+        if ($('#rootDirs option:selected').length) {
+            setDefault($('#rootDirs option:selected').attr('id'));
```

Comment:
Then, pls start a new PR. Because first time I also changed the read attr to prop. And that totally broke Medusa. Because on some of them, it gave back undefined.